kratos/utilities/normal_calculation_utils.h

```diff
@@ -90,15 +141,14 @@ class KRATOS_API(KRATOS_CORE) NormalCalculationUtils
      * @brief Calculates the area normal (vector oriented as the normal with a dimension proportional to the area).
      * @details This is done on the base of the Conditions provided which should be  understood as the surface elements of the area of interest.
      * @param rModelPart ModelPart of the problem. Must have a set of conditions defining the "skin" of the domain
-     * @param dimension Spatial dimension (2 or 3)
+     * @param Dimension Spatial dimension (2 or 3)
      * @note Use this fuction instead of its overload taking a Conditions array for MPI applications, as it will take care of communication between partitions.
      */
     void CalculateOnSimplex(
         ModelPart& rModelPart,
-        const std::size_t Dimension
+        const std::size_t Dimension = 0
```

Comment:
if this has to be 2 or 3 then the default cannot be 0!
```suggestion
        const std::size_t Dimension
```

Reply:
0 means that it must be retrieved from DOMAIN_SIZE

Reply:
can you then please add a second function that calls this one with the dimension extracted from the DOMAIN_SIZE, such that the usage is crystal clear